docs/examples/openui-crewai/frontend/src/styles.css

```
@@ -0,0 +1,33 @@
+:root {
+  color: #201b2d;
+  background: #f6f4fb;
+  font-family: Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont,
+    "Segoe UI", sans-serif;
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)